public/javascript/main.js

```diff
@@ -479,3 +479,8 @@ $(function(){
 
   }
 });
+
+function getQueryParam(key) {
```

Comment:
is this something you wrote yourself, or is it taken from somewhere else? (and if the latter should it have attribution?)